extensions/guacamole-auth-jdbc/modules/guacamole-auth-jdbc-base/src/main/java/org/apache/guacamole/auth/jdbc/JDBCAuthenticationProviderService.java

```diff
@@ -73,8 +80,22 @@ public ModeledUserContext getUserContext(AuthenticationProvider authenticationPr
 
         // Retrieve user account for already-authenticated user
         ModeledUser user = userService.retrieveUser(authenticationProvider, authenticatedUser);
-        if (user == null)
-            return null;
+        if (user == null) {
```

Comment:
That's a different method, isn't it? org.apache.guacamole.auth.jdbc.AuthenticationProviderService makes no mention of this that I can see.